samples/Makefile.am

```diff
@@ -9,57 +9,24 @@ samplessysconfdir = $(sysconfdir)
 
 samplesconf_SCRIPTS = \
 	swtpm-create-tpmca \
-	swtpm-create-user-config-files
+	swtpm-create-user-config-files \
+	swtpm-localca
 
 samplessysconf_DATA = \
 	swtpm-localca.conf \
 	swtpm-localca.options \
 	swtpm_setup.conf
 
-noinst_HEADERS = \
-	swtpm_localca.h \
-	swtpm_localca_utils.h
-
-bin_PROGRAMS = \
-	swtpm_localca
-
```

Comment:
`swtpm_localca` is now built by its own makefile in `src/swtpm_localca/`, like all the other compiled programs.